hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/WriteOperationHelper.java

```diff
@@ -590,10 +593,9 @@ public void uploadObject(PutObjectRequest putObjectRequest,
       PutObjectOptions putOptions)
       throws IOException {
 
-    retry("Writing Object",
-        putObjectRequest.getKey(), true,
-        withinAuditSpan(getAuditSpan(), () ->
-            owner.putObjectDirect(putObjectRequest, putOptions)));
+    // the transfer manager is not involved; instead it is directly
+    // PUT.
+    putObject(putObjectRequest, putOptions, null);
```

Comment:
Why the change? Is it a change?

The JavaDoc is a bit confusing. It suggests this method might use transfer manager, but we don't. Looking at `WriteOperations`, its JavaDoc says "PUT an object directly (i.e. not via the transfer manager).", which conflicts with this methods javadoc.

Reply:
i was actually stuck yesterday trying to work out why there were two duplicate methods. I think at one point the xfer manager was involved in one, but not the other. We could actually cut one entirely, which I think I will do here